lib/vmpooler/api/v1.rb

```diff
@@ -375,8 +390,16 @@ def atomically_allocate_vms(payload)
       content_type :json
       result = { 'ok' => false }
 
-      if jdata and !jdata.empty?
-        result = atomically_allocate_vms(jdata)
+      if jdata
+        empty = jdata.delete('empty')
+        invalid = jdata.delete('invalid')
+        statsd.increment(statsd_prefix + '.checkout.empty', empty) if !empty.nil?
```

Comment:
Why would you not use unless here instead of if not? (Is that rubocop or something)